desktop/ui/src/main/java/org/datacleaner/panels/AddDatastorePanel.java

```diff
@@ -68,7 +69,7 @@ public AddDatastorePanel(final DatastoreCatalog datastoreCatalog,
             final ServerInformationCatalog serverInformationCatalog, final DatabaseDriverCatalog databaseDriverCatalog,
             final DCModule dcModule,
             final DatastoreSelectedListener datastoreSelectedListener, UserPreferences userPreferences,
-            boolean showExistingDatastoresButton) {
+            boolean showExistingDatastoresButton, WindowContext wizardContext) {
```

Comment:
Since you have the dcModule here, does it really make sense to propagate the WindowContext (Which you call wizardContext for some reason... A DI habit, maybe? :smile:) all the way up? You could just create an injector here instead.